server/main.coffee

```diff
@@ -0,0 +1,76 @@
+Meteor.startup ->
+
+SPARQurL = 'http://10.0.2.230:3030/dataset'
+
+Meteor.methods(
+  'SPARQLquery': (query) ->
```

Comment:
We shouldn't use a method that takes arbitrary queries as it gives the client full access to the DB.